### PR TITLE
Adds armored tophat and tuxedo crate -- Yclat

### PR DESCRIPTION
**Pull request by SonixApache**
Finishes the armored tophat. Also adds an armored tuxedo, and puts both in a new contraband crate for cargo. Made by Yclat on the discord.

### CODE REVIEW

File: code/modules/clothing/head/misc.dm [hunk 1]
```diff
@@ -80,11 +80,12 @@
 	body_parts_covered = FULL_HEAD|BEARD
 	item_state="cueball"
 
-/obj/item/clothing/head/that
-	name = "sturdy top-hat"
-	desc = "It's an amish looking armored top hat."
+/obj/item/clothing/head/athat
+	name = "Armored top-hat"
+	desc = "It's an amish looking top hat. This one looks sturdier"
```
Comment (Kurfursten) [inline code/modules/clothing/head/misc.dm]: add .

File: html/changelogs/Yclat.yml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+#   rscadd (general adding of nice things)
+#   imageadd
+#   imagedel
+#   spellcheck (typo fixes)
+#   experiment
+#   tgs (TG-ported fixes?)
+#################################
+
+# Your name.
+author: Ghost of N3X15
```
Comment (Kurfursten) [inline html/changelogs/Yclat.yml]: the memes jack

Comment (SonixApache) [reply html/changelogs/Yclat.yml]: whoops

File: code/modules/clothing/head/misc.dm [hunk 3]
```diff
@@ -80,11 +80,12 @@
 	body_parts_covered = FULL_HEAD|BEARD
 	item_state="cueball"
 
-/obj/item/clothing/head/that
-	name = "sturdy top-hat"
-	desc = "It's an amish looking armored top hat."
+/obj/item/clothing/head/athat
+	name = "Armored top-hat"
```
Comment (ComicIronic) [inline code/modules/clothing/head/misc.dm]: "armored", it's not a proper noun.

Comment (SonixApache) [reply code/modules/clothing/head/misc.dm]: Keep forgetting it shows up in the center of a sentence.

File: code/modules/clothing/head/misc.dm [hunk 4]
```diff
@@ -80,11 +80,12 @@
 	body_parts_covered = FULL_HEAD|BEARD
 	item_state="cueball"
 
-/obj/item/clothing/head/that
```
Comment (Intigracy) [inline code/modules/clothing/head/misc.dm:83]: make a subtype of that (/obj/item/clothing/head/that/armor), the current change completely removes regular tophats (which is going to cause compile issues on the maps)

File: code/modules/clothing/head/misc.dm [hunk 5]
```diff
@@ -80,11 +80,12 @@
...
-	name = "sturdy top-hat"
-	desc = "It's an amish looking armored top hat."
+/obj/item/clothing/head/athat
+	name = "armored top-hat"
+	desc = "It's an amish looking top hat. This one looks sturdier."
 	icon_state = "tophat"
-	item_state = "that"
+	item_state = "athat"
```
Comment (Intigracy) [inline code/modules/clothing/head/misc.dm]: don't change this line if you're not adding a new tophat icon

File: code/modules/clothing/head/misc.dm (hunk 4 above)
Comment (SonixApache) [reply code/modules/clothing/head/misc.dm:83]: There's a second 'that' at line 24 -- this was always supposed to be an armored top hat, old coders just didn't implement it right.

File: code/modules/clothing/head/misc.dm (hunk 5 above)
Comment (SonixApache) [reply code/modules/clothing/head/misc.dm]: good point, fixed

File: code/modules/clothing/head/misc.dm (hunk 4 above)
Comment (Intigracy) [reply code/modules/clothing/head/misc.dm:83]: You should still make it a subtype of that.

Comment (SonixApache) [reply code/modules/clothing/head/misc.dm:83]: will-do

**Comment by Intigracy** [comment]
Now all that's left is for people to argue about the balance of the armor.

**Review by Intigracy — APPROVED** [review]
code is fine, now for people to argue about the balance part

**Comment by SonixApache** [comment]
Currently not spawned anywhere, Yclat wanted to add it to the standard formalwear closet and I suggested making an armored tuxedo. That way we'd have a set to put into an 'armored formalwear' closet.

File: code/modules/clothing/head/misc.dm (hunk 3 above)
Comment (ghost) [reply code/modules/clothing/head/misc.dm]: Armoured. :^)

**Comment by Arthurdentist** [comment]
can we have this hat next? https://www.youtube.com/watch?v=iHFqDjzkyQQ

**Comment by SonixApache** [comment]
Revision; now adds a top hat, tuxedo and a contraband wardrobe to store them in. (and fixes all the line comments)

**Comment by SonixApache** [comment]
Tested, works fine.
http://puu.sh/rn0mY/22b84abeed.png
Does this still need fixing, @clusterfack ?

**Pull request merged**